docs/htdocs/info/docs/tools/vep/index.html

```diff
@@ -17,29 +17,32 @@
 
   <div style="float:right"><img src="/img/vep_logo.png"/></div>
 	
-  <h1 id="top"><span style="color:#006">Ensembl Variant Effect Predictor (VEP)</span></h1>
+  <h1 id="top"><span style="color:#006">Ensembl Variant Effect Predictor (Ensembl VEP)</span></h1>
   <hr/>
 
   <p>
-    <span style="color:#006;font-weight:bold">VEP determines the effect of your variants</span> (SNPs, insertions,
-    deletions, CNVs or structural variants) on genes, transcripts, and
+    <span style="color:#006;font-weight:bold">Ensembl VEP predicts the effect of your variants</span> (SNPs, insertions,
+    deletions, CNVs or structural variants) on gene transcripts and
```

Comment:
```suggestion
    deletions, CNVs or structural variants) on genes, transcripts and
```

I think the update here was un-intentional?